### DISCUSSION TITLE
Support for Asobo Studio v1.2000.77.11 file formats?

### DISCUSSION THREAD

**Discussion by lorenzsandi** [bartlomiejduda/Tools#17] [Ideas]
Want to see support added for the v1.2000.77.11 version of the Asobo Studio tool that Monopoly Plus used, mainly for it's .D36, .L36, and .N36 files.

**Comment by bartlomiejduda** [comment]
Sorry, but I don't plan to support any Asobo formats in the future, as mentioned in #14.
